templates/x-whom.conf.j2

```diff
@@ -5,6 +5,6 @@
 add_header "X-Whom" "{{ ansible_eth0.ipv4.address }}";
 {% elif ansible_enp0s8 is defined %}
 add_header "X-Whom" "{{ ansible_enp0s8.ipv4.address }}";
-{% else %}
+{% elif ansible_default_ipv4 is defined %}
```

Comment:
Thanks for adding this.

At the risk of rabbit trailing too much, I don't really know how this file would ever be used unless Ansible was run on each of the web servers individually. Otherwise, this will only get written to once and all created instances will just have this statically defined, which defeats the entire purpose of this conf.

Reply:
So you want this gone?

Reply:
I think it's fine to leave in. If anything, we could use this file in conjunction with Ansible to set these values after the servers have been instantiated.